service/src/main/resources/application.yml

```diff
@@ -32,9 +32,9 @@ spring:
     password: ${springMailPassword:"none"}
     port: 465
     protocol: 'smtps'
-    dev-mode: true
```

Comment:
Is this flag from spring mail or it is made by you?

Reply:
by me it used for decide which version of `MailService` use (dev or prod)